Makefile

```diff
@@ -19,3 +19,10 @@ verify: verify-scripts
 update-scripts:
 	hack/update-apigen.sh
 .PHONY: update-scripts
+
+ensure-golangci-lint:
```

Comment:
just one nit: can we make this part of the verify-scripts target as well? I would not like to run multiple linting and verification steps